### PR TITLE
tdb - update to version 1.4.7

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

File: components/library/tdb/Makefile [hunk 1]
```diff
@@ -11,40 +11,50 @@
...
+BUILD_BITS =                    64
+BUILD_STYLE =                   waf
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= tdb
-COMPONENT_VERSION= 1.3.12
+COMPONENT_VERSION= 1.4.7
 COMPONENT_REVISION= 1
 COMPONENT_SUMMARY= Trivial Database Library
```
Comment (AndWac) [inline components/library/tdb/Makefile:24]: Remove COMPONENT_REVISION

File: components/library/tdb/Makefile [hunk 2]
```diff
@@ -11,40 +11,50 @@
...
+CONFIGURE_OPTIONS += --bindir=$(CONFIGURE_BINDIR)
+CONFIGURE_OPTIONS += --sbindir=$(CONFIGURE_SBINDIR)
 CONFIGURE_OPTIONS += --mandir=$(CONFIGURE_MANDIR)
-CONFIGURE_OPTIONS += --libdir=$(CONFIGURE_LIBDIR.$(BITS))
-CONFIGURE_OPTIONS += --without-gettext
-CONFIGURE_OPTIONS += --disable-python
+CONFIGURE_OPTIONS += --libdir=$(CONFIGURE_LIBDIR)
 CONFIGURE_OPTIONS += --disable-rpath
 
 COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
```
Comment (AndWac) [inline components/library/tdb/Makefile]: COMPONENT_TEST_MASTER = results-all.master is the default now. You can remove this line.

File: components/library/tdb/Makefile [hunk 3]
```diff
@@ -11,40 +11,50 @@
 
 #
 # Copyright 2017 Alexander Pyhalov
+# Copyright (c) 2022, Friedrich Kink
 #
 
+BUILD_BITS =                    64
+BUILD_STYLE =                   waf
```
Comment (AndWac) [inline components/library/tdb/Makefile:18]: Are you sure that there arent't any32 bit consumerw anymore?

File: components/library/tdb/Makefile (hunk 1 above)
Comment (fritzkink) [reply components/library/tdb/Makefile:24]: oh sorry I have simply overseen this one.

**Comment by AndWac** [comment]
libcanberra is 32 and 64 bit. So you cannot simply make tdb 64 bit only. Either keep also 32 bit or make libcanberra (and its dependencies) 64 bit only, too.

**Pull request merged**